packages/alfa-rules/src/sia-r60/rule.ts

```diff
@@ -0,0 +1,91 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Element, Namespace } from "@siteimprove/alfa-dom";
+import { Refinement } from "@siteimprove/alfa-refinement";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { Device } from "@siteimprove/alfa-device";
+
+import { expectation } from "../common/expectation";
+import { hasNonEmptyAccessibleName } from "../common/predicate/has-non-empty-accessible-name";
+import { hasRole } from "../common/predicate/has-role";
+import { isIgnored } from "../common/predicate";
+
+const { isElement, hasNamespace } = Element;
+const { not } = Predicate;
+const { and } = Refinement;
+
+export default Rule.Atomic.of<Page, Element>({
+  uri: "https://alfa.siteimprove.com/rules/sia-r60",
+  evaluate({ device, document }) {
+    return {
+      applicability() {
+        return document
+          .descendants({ flattened: true, nested: true })
+          .filter(isElement)
+          .filter(
+            and(
+              hasNamespace(Namespace.HTML),
+              hasRole(device, (role) => role.is("group")),
+              (group) =>
+                group.descendants({ flattened: true, nested: true }).count(
+                  and(
+                    isElement,
+                    not(isIgnored(device)),
+                    isRole(device),
+                    (ancestorRole) =>
+                      ancestorRole
+                        .closest(
+                          and(
+                            isElement,
+                            hasRole(device, (role) => role.is("group"))
+                          ),
+                          { flattened: true }
+                        )
+                        .get()
```

Comment:
```suggestion
                        // The  closest group exists because we are looking at descendants of a group.
                        .get()
```
For our own later sanity 😄